src/genie/libs/parser/iosxe/tests/ShowApDot115GhzSummary/cli/equal/golden_output_expected.py

```diff
@@ -0,0 +1,184 @@
+expected_output = {
```

Comment:
schema looks good

Reply:
please ignore. same added to https://github.com/CiscoTestAutomation/genieparser/pull/252